LaunchDarklyIosBinding/LaunchDarklyIosBinding.csproj

```diff
@@ -40,24 +39,21 @@
     <Reference Include="System" />
     <Reference Include="Xamarin.iOS" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
   <ItemGroup>
     <ObjcBindingApiDefinition Include="ApiDefinition.cs" />
+    <Compile Include="library\libDarkly.linkwith.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <ObjcBindingCoreSource Include="StructsAndEnums.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ObjcBindingCoreSource Include="Structs.cs" />
+    <BundleResource Include="packages.config" />

```

Comment:
I guess this csproj is a PCL and not a .NET Standard project. I think that's ok but I wanted to double check that we're fine to reference the PCL nuget from the other projects. @chrisr-ifit ?